src/lambdas/breeds/breeds-get.ts

```diff
@@ -0,0 +1,59 @@
+import fetch from 'node-fetch'
+import { Response, ErrorResponse } from '../types'
+
+export interface DogBreedsListResponse extends Response {
+  body: string[]
+}
+
+interface DogBreedsObject {
+  message: {
+    [key: string]: string[]
+  }
+  status: string
+}
+
+function reduceBreedResponse(
+  collector: string[],
+  [breed, subBreeds]: [string, string[]],
+): string[] {
+  if (subBreeds && subBreeds.length) {
+    return [...collector, ...subBreeds.map((subBreed) => `${subBreed} ${breed}`)]
+  }
+  return [...collector, breed]
+}
+
+export async function handler(): Promise<DogBreedsListResponse | ErrorResponse> {
+  try {
+    // this could be wrapped in a Promise.race to force a timeout (lambda should stop execution)
+    // or an AbortController could be used: https://github.com/node-fetch/node-fetch#request-cancellation-with-abortsignal
```

Comment:
Could you go ahead and implement one of these?  See my note on the `408` above in the test file. That should clean up the test and return the actual Error fetch throws.

Reply:
Implemented the AbortController approach

Reply:
5e3980d735ceb0b1af9db891df6c0bac923bfac0